crml/staking/src/tests.rs

```diff
@@ -208,45 +207,31 @@ fn rewards_should_work() {
 			},
 		);
 
-		<Payee<Test>>::insert(&2, RewardDestination::Stash);
-		assert_eq!(Staking::payee(2), RewardDestination::Stash);
-		assert_eq!(Staking::payee(11), RewardDestination::Account(10));
+		assert_eq!(Staking::payee(&2), RewardDestination::Stash);
+		assert_eq!(Staking::payee(&11), RewardDestination::Controller);
```

Comment:
~change assertion back to `RewardDestination::Account(10)` re discussion above (RewardDestination::Controller is phased out)~

Reply:
Sorry which discussion?

Reply:
check the thing we actually store
```suggestion
		assert_eq!(<<Staking as Trait>::Rewarder as HandlePayee>::payee(&11), 11)
```

Reply:
Done